readme.md

```diff
@@ -59,26 +73,9 @@
     - [Customize the Close header's button icon](#customize-the-close-headers-button-icon)
     - [Hide the modal background](#hide-the-modal-background)
 
```

Comment:
Please make the following change above:

Line 51:
"DxCalender" is misspelled. It should be:

`- [DxCalendar](#dxcalendar)`